src/k8s-engine/lima.ts

```diff
@@ -557,7 +557,18 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
       }
 
       // Start the VM; if it's already running, this does nothing.
-      await this.lima('start', '--tty=false', await this.isRegistered ? MACHINE_NAME : CONFIG_PATH);
+      try {
+        await this.lima('start', '--tty=false', await this.isRegistered ? MACHINE_NAME : CONFIG_PATH);
+      } finally {
+        // Symlink the logs (especially if start failed) so the users can find them
+        const machineDir = path.join(LIMA_HOME, MACHINE_NAME);
+        const fileNames = ['serial.log', 'ha.stdout.log', 'ha.stderr.log'];
```

Comment:
Maybe search `*.log`, as we may add more log files.

Reply:
Thanks for the suggestion! Changed in a69ec3d6809b7bded8b21753d3e56b41c68ab01b.